src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const Robot = {
```

Comment:
this is just a object literal, name should start from a small letter (on the other hand, classes names are capitalized)
```suggestion
  const robot = {
```